### PR TITLE
feat(template): Add tests for templates

### PR DESCRIPTION
**Pull request by udhadpa**
Add tests for Templates : Desktop and Mobile.
@JasonLantz 
@umahaea 
@StommePoes 
@wyseguyonline

### CODE REVIEW

File: src/main/java/elements/fixtures/templates.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+</head>
+<body>
+
+<!--  Single Column -->
+<p>
+<h4>Single Column</h4>
+<div id="template-single" class='pe-template__single' style='background:magenta;'>
+    <h6 style='background:cyan;text-align:center;height:40px;'>Single Column - Content </h6>
+</div>
+</p>
```
Comment (StommePoes) [inline src/main/java/elements/fixtures/templates.html]: This batch of code shouldn't be wrapped in a p element-- p elements aren't allowed to contain *block* elements like divs or headings. This means the browser will use its Error rendering to deal with it. 

If you change it to a div, it'll be ok.

**Comment by StommePoes** [comment]
Other than my HTML comment, this seems ok to me.

File: src/main/java/elements/fixtures/templates.html (hunk 1 above)
Comment (udhadpa) [inline src/main/java/elements/fixtures/templates.html]: @StommePoes : Hi Mallory,
I have removed the <p>tag. I wanted to group and space out those elements,but it was not necessary.
Thanks for the valuable feedback.

**Comment by umahaea** [comment]
looks good to me..

File: src/main/java/elements/fixtures/templates.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        <h6 style='background:lightblue;text-align:center;height:40px;'>Directly Adjoin - SideBar </h6>
+    </div>
+</div>
+<div id="template-single-bottom" class='pe-template__single' style='background:LightSkyBlue;'>
+    <h6 style='background:SteelBlue;text-align:center;height:40px;'>Directly Adjoin - Content</h6>
+</div>
+
+<!-- Small Spacing -->
+<h4>Standard Spacing</h4>
+<div id="template-smallgap" class='.pe-template__small-gap'>
```
Comment (JasonLantz) [inline src/main/java/elements/fixtures/templates.html]: take the dot off the class

File: src/main/java/elements/fixtures/templates.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        </div>
+    </div>
+    <div id="template-single-smallgap-bottom" class='pe-template__single' style='background:LightCyan;'>
+        <h6 style='background:lightblue;text-align:center;height:40px;'>Small Space - Content </h6>
+    </div>
+</div>
+
+<!-- Large Spacing -->
+<h4>Large spacing </h4>
+<div id="template-largegap" class='.pe-template__large-gap'>
```
Comment (JasonLantz) [inline src/main/java/elements/fixtures/templates.html]: same dot on class.

File: src/main/java/elements/fixtures/templates.html (hunk 2 above)
Comment (StommePoes) [inline src/main/java/elements/fixtures/templates.html]: nice catch

File: src/main/java/elements/fixtures/templates.html (hunk 3 above)
Comment (udhadpa) [inline src/main/java/elements/fixtures/templates.html]: Fixed the "." in the templates.html

**Comment by udhadpa** [comment]
Merging the branch

**Pull request merged**